Document/0x06a-Platform-Overview.md

```diff
@@ -28,18 +28,22 @@ The [iOS security architecture](https://www.apple.com/business/docs/iOS_Security
 
 #### Hardware Security
 
-The iOS security architecture makes heavy use of hardware-based security features that enhance overall performance and security. Each device comes with two built-in AES 256-bit keys, UID and GID, fused/compiled into the application processor and Secure Enclave during manufacturing. There is no way to directly read these keys through software or debugging interfaces such as JTAG. Encryption and decryption operations are performed by hardware AES crypto-engines with exclusive access to the keys.
+iOS security architecture makes heavy use of hardware-based security features that enhance overall performance and security. Each iOS device comes with two built-in AES 256-bit keys – GID and UID – fused and compiled into the application processor and Secure Enclave during manufacturing. There is no direct way to read these keys by a software or debugging interfaces such as JTAG. Encryption and decryption operations are performed by hardware AES crypto-engines having exclusive access to these keys.

```

Comment:
> *The* iOS security

I think leaving "The" here would be better.

> read these keys by *a* software

Software is an uncountable noun, so you can't have *a* software.